alg/teca_2d_component_area.cxx

```diff
@@ -133,6 +133,8 @@ void teca_2d_component_area::get_properties_description(
         TECA_POPTS_GET(int, prefix, contiguous_component_ids,
             "when the region label ids start at 0 and are consecutive "
             "this flag enables use of an optimization (0)")
+        TECA_POPTS_GET(short, prefix, background_id,
+            "the label id that corresponds to the background (-1)")
```

Comment:
this default differs from the default set on line 112 - is the idea here that if a user overrides this on the command line, we're assuming that they want to set it to something other than 0?

I'm mainly checking to be sure that -1 isn't a typo.

Reply:
Now I'm wondering if initializing `background_id(0)` is a typo, since it appears this conflicts with the comments provided in the header file:
```
     // By default
     // this is set to -1 to indicate that the value should be obtained from the
     // metadata key `background_id`
```

Reply:
yes, good catch, it should be -1. I'll make that change now.